src/main/java/com/academia/library/model/Authority.java

```diff
@@ -0,0 +1,29 @@
+package com.academia.library.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
You don't have to implement NoArgsConstructor without another constructor
Please repeat how constructor works in Java

Reply:
Ok